apps/dash-mapd-demo/app.py

```diff
@@ -0,0 +1,649 @@
+import dash
+import dash_table
+import dash_core_components as dcc
+import dash_html_components as html
+import plotly.graph_objs as go
+
+import numpy as np
+import pandas as pd
+from dash.dependencies import State, Input, Output
+from dash.exceptions import PreventUpdate
+import pymapd
+import datetime
+from datetime import datetime as dt
+import os
+
+app = dash.Dash(__name__)
+server = app.server
+
+app.config.suppress_callback_exceptions = True
+
+# Get credentials from env
+user = "mapd"
+password = "HyperInteractive"
+db_name = "mapd"
+
+if "DASH_APP_NAME" in os.environ:
+    host = os.environ.get("DB_HOST")
+else:
+    host = "localhost"
+
+table = "flights_2008_7M"
+
+
+# Connect to omnisci server
+def db_connect():
+    try:
+        connection = pymapd.connect(
+            user=user, password=password, host=host, dbname=db_name
+        )
+
+        if table not in connection.get_tables():
+            print("Table {} not found in this database, please load sample data.")
+
+        return connection
+
+    except Exception as e:
+        print("Error connection to db : {}".format(e))
+
+
+con = db_connect()
+print(con.get_tables())
+
+
+def build_banner():
+    return html.Div(
```

Comment:
Please add a docstring for this function too (if you have docstrings for other functions)!